javascript/lib/do_username.js

```diff
@@ -74,3 +17,12 @@ module.exports.generate = (maxSize = 30) => {
 
     return combineUsername(maxSize, descriptor, color, noun);
 };
+
+module.exports = {
+    SEA_LIST,
+    SEA_CREATURES,
+    DESCRIPTORS,
+    COLORS,
+    CREATURE_DESCRIPTORS,
```

Comment:
We're not exporting the full set of constants here

Reply:
oops